js/views/components/moderators/Card.js

```diff
@@ -86,7 +91,10 @@ export default class extends BaseVw {
   }
 
   get hasValidCurrency() {
-    return anySupportedByWallet(this.modCurs);
+    const isFeeFiat = this.fixedFeeCur && isFiatCur(this.fixedFeeCur);
+    const isFeeCrypto = this.fixedFeeCur && anySupportedByWallet([this.fixedFeeCur]);
+    const validFeeCur = isFeeFiat || isFeeCrypto;
```

Comment:
I didn't see an existing method for checking if a currency code was a known fiat or crypto, if there is one that could be swapped in here.